src/Turn.js

```diff
@@ -1,33 +1,39 @@
+import Puzzle from './Puzzle';
+import Round from './Round';
+import Turn from './Turn';
+import Wheel from './Wheel';
 class Turn {
-  constructor(player, round) {
-    this.player = player;
+  constructor(round, player) {
     this.round = round;
+    this.player = player;
+    this.lettersRemaining = Puzzle.correctAnswer.split() || [];
   }
 
-
   spinwheel() {
-    //wheel.findWheelPrize()
-    //checkCurrentPlayer();
+    Round.wheel.chooseWheelPrize()
```

Comment:
Interesting that you are calling the `chooseWheelPrize()` from `Round.wheel`. The capital "Round" is what is confusing me. Here you want to call `this.round` instead of `Round` because you don't want to access the class definition (capital Round). Access the instance that is shared with Turn (`this.round`).